axelor-human-resource/src/main/java/com/axelor/apps/hr/service/KilometricService.java

```diff
@@ -210,8 +211,8 @@ public BigDecimal computeKilometricExpense(ExpenseLine expenseLine, Employee emp
         price = price.add(max.subtract(min).multiply(rule.getRate()));
       }
     }
-
-    return price.setScale(2, RoundingMode.HALF_UP);
+    return price.setScale(
+        appBaseService.getAppBase().getNbDecimalDigitForUnitPrice(), RoundingMode.HALF_UP);
```

Comment:
we have to keep the hard coded `2` since the field `totalAmount` is only on 2 digits